src/loginscreen.js

```diff
@@ -0,0 +1,104 @@
+const username = document.createElement("input");
+
+
+function loginScreen () {
+    const content = document.getElementById("content");
+    const loginContent = document.createElement("div");
+    loginContent.setAttribute("id", "loginContent");
+    content.appendChild(loginContent);
+
+    const form = document.createElement("form");
+    loginContent.appendChild(form);
+    form.appendChild(username);
+    username.placeholder = "Username";
+    let yourName = username.value;
+
+    const loginButton = document.createElement("button");
+    loginButton.textContent = "Login";
+    loginContent.appendChild(loginButton);
+
+    loginButton.addEventListener("click", function(){
+
+        content.removeChild(loginContent);
+        getPreviousMessages();
+        messagePage();
+
+    })
+
+    async function getPreviousMessages(){
+        for (let i = 0; i < 8; i++) {
```

Comment:
API call returns a list of messages already, you do not need to call the API 8 times.